doc/source/whatsnew/v0.24.2.rst

```diff
@@ -53,6 +53,7 @@ Bug Fixes
 
 - Bug in :meth:`~pandas.core.groupby.GroupBy.transform` where applying a function to a timezone aware column would return a timezone naive result (:issue:`24198`)
 - Bug in :func:`DataFrame.join` when joining on a timezone aware :class:`DatetimeIndex` (:issue:`23931`)
+- Bug in groupby.idxmin() that returns DateTime values for DataTime columns (:issue:`25444`)
```

Comment:
not sure what you mean here, these are not valid things 'DateTime' or 'DataTime'

Reply:
what about: "Functions idxmin and idxmax on :meth:`~DataFrame.groupby` works correctly with  datetime64 data."?

Reply:
```suggestion
- Bug in :meth:`GroupBy.idxmin() <pandas.core.groupby.GroupBy.idxmin>` and :meth:`GroupBy.idxmax() <pandas.core.groupby.GroupBy.idxmax>` where datetime columns would incorrectly return datetime values instead of ints (:issue:`25444`)
```

Reply:
Can you also move to 0.25 at this point?

Reply:
thank you for suggestion. Reformulated and moved to 0.25 in https://github.com/pandas-dev/pandas/pull/25531/commits/510b22135575ee6ebedd559a6d5ecb69e4b9a376